modules/internal/ChapelSyncvar.chpl

```diff
@@ -43,7 +43,7 @@ module ChapelSyncvar {
   ************************************* | ************************************/
 
   //
-  // The following types are OK for full empty types (sync/single)
+  // The following types are OK for full empty types (sync)
```

Comment:
I think this shouldn't be plural any more, right?  It should just talk about `sync` instead of in the more general way